cypress/support/pages/main-page.ts

```diff
@@ -0,0 +1,20 @@
+import { CATEGORY } from "../../elements/category";
+import { MAIN_PAGE } from "../../elements/main-page";
+import { NAVIGATION } from "../../elements/navigation";
+import { SHARED } from "../../elements/shared";
+
+export function selectNotEmptyCategory() {
+  cy.get(MAIN_PAGE.categorySection)
+    .find(SHARED.productsList)
+    .first()
+    .parents(MAIN_PAGE.categorySection)
+    .children(MAIN_PAGE.categoryName)
+    .invoke("text")
+    .then((categoryTitle) => {
```

Comment:
Wouldn't be better, to get not empty category via API?

Reply:
I've talked with @michalina-graczyk  and there is a decision to not use the API requests.